scripts/irods/test/test_iadmin.py

```diff
@@ -485,6 +485,20 @@ def test_iadmin_mkuser(self):
         self.admin.assert_icommand("iadmin mkuser lando invalid_user_type",
                                    'STDERR_SINGLELINE', "CAT_INVALID_USER_TYPE")  # should be rejected
 
+    @unittest.skipIf(IrodsConfig().version_tuple < (4, 3, 2), "Only applies to 4.3.2 and above.")
+    def test_iadmin_rmuser_rmgroup_report_error_on_mismatch_entity_type__issue_7380(self):
+        self.admin.assert_icommand("iadmin mkuser testuser rodsuser")
+        self.admin.assert_icommand("iadmin mkgroup testgroup")
```

Comment:
`testuser` and `testgroup` are very generic. Consider including the issue number in the names so there's less of a chance for colliding with other tests.

You could store the test user/group names in variables and use those throughout the test too. That would make it easy to update the test names later if needed.